media-gfx/povray/povray-3.7.0.8.recipe

```diff
@@ -77,14 +75,24 @@ BUILD()
 {
 	cd unix
 	./prebuild.sh
+	autoupdate
 	cd ..
 
-	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
 	export LDFLAGS="-lbsd -lnetwork"
 
-	runConfigure ./configure \
-		COMPILED_BY="HaikuPorts" \
-		--with-boost-libdir=`finddir B_SYSTEM_LIB_DIRECTORY`
+	if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
+		export CXXFLAGS="-march=pentium -mtune=generic -std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
+		runConfigure ./configure \
+			COMPILED_BY="HaikuPorts" \
+			--without-openexr \
+			--with-boost-libdir=/boot/system/lib/x86
+	fi
+	if [ $effectiveTargetArchitecture = x86_64 ]; then
+		export CXXFLAGS="-march=core2 -mtune=generic -std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
+		runConfigure ./configure \
+			COMPILED_BY="HaikuPorts" \
+			--with-boost-libdir=`finddir B_SYSTEM_LIB_DIRECTORY`
+	fi
```

Comment:
Instead of this, simply adding the option `--disable-optimiz` should take care of `-march=native`

Reply:
Fixed.